certbot/tests/main_test.py

```diff
@@ -1000,7 +1018,7 @@ def test_renew_obtain_cert_error(self):
             mock_rc.return_value = mock_lineage
             mock_lineage.configuration = {
                 'renewalparams': {'authenticator': 'webroot'}}
-            with mock.patch('certbot.main.obtain_cert') as mock_obtain_cert:
+            with mock.patch('certbot.main.renew_cert') as mock_obtain_cert:
```

Comment:
This should probably be `mock_obtain_cert`, although I care less than I do about `afa` because this name is much better.